{{cookiecutter.project_slug}}/templates/blocks/html_block.html

```diff
@@ -0,0 +1,3 @@
+<div class="block html-block">
+  {{ self.code|safe }}
```

Comment:
I just put the html_block template in for churchinwales.  Technically the `|safe` shouldn't be needed, I think, as inside the `RawHTMLBlock` it's returned via `mark_safe`...

I'd also not put a `div` block around the html, in case someone wanted / neede to do weird stuff with putting a whole bunch of other blocks inside a wrapper or something mad like that.

I'd considered adding a `is_not_full_html_fragment` safety toggle, so that by default it would test the html as being a valid full HTML fragment w/ htmllib or something, and only disable it if specifically requested, which might? stop broken code breaking the rest of the page... 

What do you think?